### PR TITLE
default the internal registry hostname to what we expect it to be, save a restart

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: pkg/operator/configobservation/images/observe_images.go [hunk 1]
```diff
@@ -39,6 +39,11 @@ func ObserveInternalRegistryHostname(genericListers configobserver.Listers, reco
 	configImage, err := listers.ImageConfigLister.Get("cluster")
 	if errors.IsNotFound(err) {
 		glog.Warningf("image.config.openshift.io/cluster: not found")
+		// if we have no cluster image config object, default the internal registry hostname to what we expect it will be
```
Comment (deads2k) [inline pkg/operator/configobservation/images/observe_images.go]: do you want this same logic down on an else on line 54?

**Comment by bparees** [comment]
after discussion with @deads2k we agree that defaulting this to anything is the wrong thing to do as it implies a registry is available when one may not be.

**Pull request closed without merging**